rh/schema.py

```diff
@@ -76,6 +129,29 @@ def perform_mutate(cls, form, info):
         return cls.mutation_success()
 
 
+class RhRentStabData(graphene.ObjectType):
+    latest_year = graphene.Int(
+        description=(
+            "The last year that the user's building had rent stabilized units. "
+            "If null, no units were found since 2007."
+        )
+    )
+    latest_unit_count = graphene.Int(
+        description=(
+            "The most recent count of rent stabilized units in user's building. "
+            "If null, no units were found since 2007."
+        )
+    )
+
+
 @schema_registry.register_session_info
 class RhSessionInfo(object):
     rental_history_info = RhFormInfo.field()
+    rent_stab_info = graphene.Field(RhRentStabData)
+
+    def resolve_rent_stab_info(self, info: ResolveInfo):
+        request = info.context
+        kwargs = request.session.get(RENT_STAB_INFO_SESSION_KEY, {})
+        if kwargs:
+            return RhRentStabData(**kwargs)
```

Comment:
This case should have test coverage... we could unify it into the test for the mutation, that is how many of my GraphQL tests work.

Reply:
Slated some time on Monday for us to work through this together!